src/utils/Resize.js

```diff
@@ -92,7 +92,7 @@ function Resize (elementId, neonView, dragHandler) {
       d3.drag()
         .on('start', resizeStart)
         .on('drag', resizeDrag)
-        .on('end', resizeEnd)
+        .on('end', resizeEnd.bind(this))
```

Comment:
Do these changes break resizing staves?

Reply:
"this" only gets called in resizeEnd when calling selectBBox, so the behaviour for resizing staves is the same